dbot/dbot/CommandModules/votingmodule.cs

```diff
@@ -94,6 +96,27 @@ public async Task Default(int movId)
             }
         }
 
+        [Command]
+        [Priority(1)]
+        public async Task Default(string mov)
+        {
+            if (_votingService.votingOpen())
+            {
+                var noms = _nominationsService.getNominations();
+                var myNomObj = noms.FirstOrDefault(x => x.movName.Equals(mov));
```

Comment:
Instead of FirstOrDefault when you expect 1 result you can do .Single() and it will throw if there is none

Reply:
that would be better

Reply:
If you want to change it i think it would be
var myNomObj = noms.Where(x => x.movName.Equals(mov)).Single();